FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -902,7 +907,7 @@ void FireRenderContext::render(bool lock)
 		}
 
 		m_restartRender = false;
-		m_startTime = clock();
+		m_renderStartTime = clock();
```

Comment:
why not use chrono?

Reply:
because clock() is used in almost all the places. to keep one style at the moment